app/views/user_mailer/_work_info.text.erb

```diff
@@ -0,0 +1,22 @@
+<% # expects work %>
+<%= work_metadata_label(Work.human_attribute_name("chapter_total_display"))%><%= work.chapter_total_display %>
+<%= work_tag_metadata(work.fandoms) %>
+<%= work_tag_metadata(work.ratings) %>
+<%= work_tag_metadata(work.archive_warnings) %>
+<% unless work.relationship_string.blank? %>
+<%= work_tag_metadata(work.relationships) %>
+<% end %>
+<% unless work.character_string.blank? %>
+<%= work_tag_metadata(work.characters) %>
+<% end %>
+<% unless work.freeform_string.blank? %>
+<%= work_tag_metadata(work.freeforms) %>
+<% end %>
+<% unless work.series.count.zero? %>
+<%= Series.model_name.human(count: work.series.count) %><%= raw to_plain_text(series_list_for_feeds(work)) %>
+<% end %>
+<% unless work.summary.blank? %>
+
+<%= work_metadata_label(Work.human_attribute_name("summary")) %>
+    <%= raw to_plain_text(sanitize_field(work, :summary)) %>
```

Comment:
Should this be indented 2 spaces?

Reply:
Nope, I wanted to match the indenting from the subscription emails (I assume we went with four spaces because it's more obviously done for a pretty formatting effect than two -- two just looks kinda like we misaligned the text): 
https://github.com/otwcode/otwarchive/blob/5cd77dd6d6299fcd8b578115af460cad22ea3e78/app/views/user_mailer/batch_subscription_notification.text.erb#L31-L32